### ISSUE TITLE
How can I retriever the credited transactions?

### ISSUE THREAD

**Issue by goromachine** [Mangopay/mangopay2-python-sdk#215]
When I use `wallet.transactions.all()`, I'm only receiving the debited transactions, but to show to the user the whole transactions history, I would need to show also those transactions in which it is the credited_user.

Why does `wallet.transactions.all()` do not get the credited transactions? And which it will be the easy way to acquire them? 

I can just go transaction ID by transaction ID and get them all, but seems that this is not the correct way because of the inefficiency of doing many requests in front of just one to get them all.

Thanks

**Comment by goromachine** [comment]
I thought that maybe I won't get them because of the pagination, so I decided to apply what people already told me on #209. So I made the following:

`transactions = wallet.transactions.all(page=1, ItemsPerPage=100)`

Even with that I still only receiving 10 out of 16 transactions. the wallet is correct because I verified the 10 triggered transactions with the first 10 transactions of that wallet.

What could I be doing wrong here? On the documentation says those two parameters are the ones needed to trigger more than 10 items per page, right? I tried with many pages and items between 10-100 but still only getting the same 10 ones.
